modules/security-context-constraints-about.adoc

```diff
@@ -52,8 +52,7 @@ effectively root on the cluster and must be trusted accordingly.
 
 [IMPORTANT]
 ====
-Do not modify the default SCCs. Customizing the default SCCs can lead to issues
-when {product-title} is upgraded.
+Do not modify the default SCCs. Customizing the default SCCs can lead to issues when {product-title} is upgraded and some of the platform pods deploy. During upgrades between some versions of {product-title}, the values of the default SCCs are reset to the default values, which discards all customizations to those SCCs.
```

Comment:
I'd maybe reorder the sentence like so:
```
Customizing the default SCCs can lead to issues when some of the platform pods get redeployed or when {product-title} is upgraded.
```
simply because the problems with platform pods are more likely and sound more urgent.

Otherwise LGTM

Reply:
Thanks!